CMakeLists.txt

```diff
@@ -682,7 +682,14 @@ elseif(CMAKE_SYSTEM_PROCESSOR MATCHES "x86_64|amd64|AMD64")
 elseif(CMAKE_SYSTEM_PROCESSOR MATCHES "x86|i386|i686")
   set(ARCH "x86")
 elseif(CMAKE_SYSTEM_PROCESSOR MATCHES "arm64.*|ARM64|aarch64")
-  set(ARCH "aarch64")
+  # If ARCH is originally detected as 64-bit, perform an additional check
```

Comment:
Is this only an Arm issue?

Reply:
it's also x86 issue but it was already fixed, see few lines above